modules/tinymce/src/themes/silver/demo/ts/demo/ButtonSetupDemo.ts

```diff
@@ -78,6 +80,126 @@ export default {
       }
     });
 
+    // This fetch function is used for the MailMerge example buttons. The
+    // flag passed in determines whether to collapse all the items into a single
+    // menu when a search pattern is present.
+    const mailMergeFetch = (collapseIfSearch: boolean) => (callback: any, fetchContext: any) => {
```

Comment:
Don't we have types we can pull in here. Even if it's a demo. It tends to rot if changes are made to the api unless we have some type checking.

Reply:
We haven't been exposing Success Callback, but I can probably do it by using `['fetch']` of MenuButton.

Reply:
- [ ] see if we can get types into Demo.

Reply:
Updated: used `ToolbarMenuButton['fetch']`